design/sketch.webidl

```diff
@@ -484,23 +484,20 @@ enum GPUInputStepMode {
 };
 
 dictionary GPUVertexAttributeDescriptor {
-    u32 shaderLocation;
-    u32 inputSlot;
     u64 offset;
     GPUVertexFormat format;
+    u32 shaderLocation;
 };
 
-dictionary GPUVertexInputDescriptor {
-    u32 inputSlot;
+dictionary GPUVertexBufferDescriptor {
     u64 stride;
     GPUInputStepMode stepMode;
+    sequence<GPUVertexAttributeDescriptor> attributes;
```

Comment:
Can we call this `attributeSet`, so it's clear it's unordered? (Or rather that the order doesn't matter)

Reply:
I wish there were a better way to describe a set here (JS has a Set...) because this name still isn't extremely clear to me. Maybe it's just because I hear terms like "descriptor set" too much.

Anyway, done, unless there are any better name suggestions.